examples/AnnotatorForClinicalDataExample.java

```diff
@@ -0,0 +1,543 @@
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.ibm.cloud.sdk.core.http.Response;
+import com.ibm.cloud.sdk.core.http.ServiceCall;
+import com.ibm.cloud.sdk.core.security.Authenticator;
+import com.ibm.cloud.sdk.core.security.BasicAuthenticator;
+import com.ibm.watson.health.acd.v1.AnnotatorForClinicalData;
+import com.ibm.watson.health.acd.v1.model.AcdCartridges;
+import com.ibm.watson.health.acd.v1.model.AcdFlow;
+import com.ibm.watson.health.acd.v1.model.AcdProfile;
+import com.ibm.watson.health.acd.v1.model.Annotator;
+import com.ibm.watson.health.acd.v1.model.Annotator.Name;
+import com.ibm.watson.health.acd.v1.model.AnnotatorFlow;
+import com.ibm.watson.health.acd.v1.model.CartridgesGetIdOptions;
+import com.ibm.watson.health.acd.v1.model.CartridgesGetOptions;
+import com.ibm.watson.health.acd.v1.model.CartridgesPostMultipartOptions;
+import com.ibm.watson.health.acd.v1.model.CartridgesPutMultipartOptions;
+import com.ibm.watson.health.acd.v1.model.Concept;
+import com.ibm.watson.health.acd.v1.model.ContainerGroup;
+import com.ibm.watson.health.acd.v1.model.CreateFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.CreateProfileOptions;
+import com.ibm.watson.health.acd.v1.model.CreateProfileOptions.Builder;
+import com.ibm.watson.health.acd.v1.model.DeleteFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.DeleteProfileOptions;
+import com.ibm.watson.health.acd.v1.model.DeleteUserSpecificArtifactsOptions;
+import com.ibm.watson.health.acd.v1.model.DeployCartridgeOptions;
+import com.ibm.watson.health.acd.v1.model.DeployCartridgeResponse;
+import com.ibm.watson.health.acd.v1.model.Flow;
+import com.ibm.watson.health.acd.v1.model.FlowEntry;
+import com.ibm.watson.health.acd.v1.model.GetAnnotatorsByIdOptions;
+import com.ibm.watson.health.acd.v1.model.GetAnnotatorsOptions;
+import com.ibm.watson.health.acd.v1.model.GetFlowsByIdOptions;
+import com.ibm.watson.health.acd.v1.model.GetFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.GetProfileOptions;
+import com.ibm.watson.health.acd.v1.model.GetProfilesOptions;
+import com.ibm.watson.health.acd.v1.model.ListStringWrapper;
+import com.ibm.watson.health.acd.v1.model.ServiceApiBean;
+import com.ibm.watson.health.acd.v1.model.ServiceStatus;
+import com.ibm.watson.health.acd.v1.model.UpdateFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.UpdateProfileOptions;
+import com.ibm.watson.health.acd.v1.util.FlowUtil;
+
+/**
+ * Sample ACD Java SDK Calls
+ */
+public class AnnotatorForClinicalDataExample {
+
+	public static String ACD_APIKEY="<YOUR_APIKEY>";
+	public static String ACD_URL="https://us-east.wh-acd.cloud.ibm.com/wh-acd/api";
+	//public static String ACD_URL="https://us-south.wh-acd.cloud.ibm.com/wh-acd/api";
+	public static String version = "2020-06-01";
+	public static String name="AnnotatorForClinicalData";
```

Comment:
Is 'name' variable at line 72 ever used?  At lines 68 and 69, should these be 'static final' (uppercase implies a constant)?